apps/frontend/src/emails/templates/moderation/ModerationThreadMessageReceived.vue

```diff
@@ -0,0 +1,63 @@
+<script setup lang="ts">
+import { Heading, Img, Link as VLink, Section, Text } from '@vue-email/components'
+
+import StyledEmail from '@/emails/shared/StyledEmail.vue'
+</script>
+
+<template>
+	<StyledEmail
+		title="New moderation message"
+		:manual-links="[
+			{ link: 'https://modrinth.com/legal/rules', label: 'Community Guidelines' },
+			{
+				link: 'https://modrinth.com/project/{project.id}/moderation',
+				label: 'Your project\'s moderation thread',
+			},
+			{ link: 'https://support.modrinth.com', label: 'Support Portal' },
+		]"
+	>
+		<Section class="mb-4 mt-2">
+			<Img
+				src="{project.icon_url}"
+				alt="project icon"
+				width="64"
+				height="64"
+				class="block h-auto"
+			/>
+		</Section>
+		<Heading as="h1" class="mb-2 text-2xl font-bold">New moderation message</Heading>
+
+		<Text class="text-base">Hi {user.name},</Text>
+
+		<Text class="text-base">
+			A new message has been posted in the
+			<VLink
+				href="https://modrinth.com/project/{project.id}/moderation"
+				class="text-green underline"
+				>moderation thread</VLink
+			>
+			for your project <b>{project.name}</b>.
+		</Text>
+
+		<Section class="bg-bg-super mb-4 mt-4 rounded-lg border border-divider pb-4 pl-4 pr-4 pt-4">
+			<Text class="m-0 text-base">
+				Log in to review the message and reply if needed. Addressing moderator feedback promptly
+				helps move your project forward.
```

Comment:
"Please [sign in to view the message](link to moderation thread again) and reply if requested. It's important to address feedback from the moderation team promptly."